core/backend/src/assets/Settings/Schemas/Workspace.Schema.json

```diff
@@ -1,53 +1,93 @@
 {
-  "$id": "http://itwinjs.org/json-schema/workspace",
-  "$schema": "http://json-schema.org/draft-07/schema",
-  "title": "Workspace settings",
-  "description": "settings for accessing data from workspaces",
-  "type": "object",
-  "groupName": "workspace",
-  "properties": {
-    "workspace/databases": {
+  "$schema": "itwin-settings/schema/1.1",
```

Comment:
Keith, what is `itwin-settings/schema/1.1` and does it even exist somewhere? If it does not exist, is intellisense (auto-complete) not available in your IDE when creating a new schema?

Reply:
I think we should at least replace `itwin-settings/schema/1.1` with an actual "meta" or "base" JSON Schema that defines what "settingDef", "typeDef", and "groupName" are, and host that JSON schema somewhere so you can reference it like "http://somewhere/itwin-settings/schema/1.1"

E.g.,
```json5
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "additionalProperties": false,
  "properties": {
    "settingDefs": {
      "type": "object",
      "description": "A setting definition"
    },
    "typeDefs": {
      "type": "object",
      "description": "A type definition that can be used in the settingDefs, e.g., 'extends': 'itwin/core/workspace/workspaceDb'"
    },
    "groupName": { // or change this to "schemaId"
      "type": "string",
      "description": "The unique id of the schema"
    }
  }
}
```

Reply:
OK, but should that also be in a local meta-meta-schema file? 

Shouldn't the descriptions of `settingDefs` and `typeDefs` be plural (e.g. "A group of Setting definitions", "A group of Type definitions that each can be used...")?

How 'bout `SchemaPrefix` instead of `groupName` (Id sounds like a number)?

Reply:
> OK, but should that also be in a local meta-meta-schema file?

As long as its accessible to everyone. One thing I've seen ppl do is reference it directly from Github that way you don't have to go through the pain of hosting it somewhere else:

```json5
// my app's settings.schema.json
{
  "$schema": "https://raw.githubusercontent.com/iTwin/itwinjs-core/master/core/backend/src/assets/Settings/Base.Schema.json",
  // ... now I get intelli-sense for the rest of my file!
  "settingsDef": {
    // ... etc 
  }
}
```

> Shouldn't the descriptions of `settingDefs` and `typeDefs` be plural (e.g. "A group of Setting definitions", "A group of Type definitions that each can be used...")?

Yes you're right

> How 'bout `SchemaPrefix` instead of `groupName` (Id sounds like a number)?

That sounds right

Reply:
Will you make these changes? I'm looking for volunteers to take over this PR...

Reply:
Yeah, I can